graylog2-web-interface/src/views/logic/searchtypes/events/EventHandler.js

```diff
@@ -31,17 +35,25 @@ export type Shapes = Array<Shape>;
 const eventsDisplayName = 'Alerts';
 const defaultColor = '#d3d3d3';
 
+const formatTimestamp = (timestamp, tz = 'UTC'): string => {
+  // the `true` parameter prevents returning the iso string in UTC (http://momentjs.com/docs/#/displaying/as-iso-string/)
+  return moment(timestamp).tz(tz || 'UTC').toISOString(true);
```

Comment:
You can use `tz ?? 'UTC'` here.

Just to let you know, I started replacing 'UTC'  in another PR (https://github.com/Graylog2/graylog2-server/pull/7836) with `AppConfig.rootTimeZone();`. I thought about replacing it here as well, but we did not merged the mentioned PR and use `UTC` in comparable cases.